fix: github action version (#71)
kubeshop/testkube-docs: docs/articles/test-workflows.md

      image: anasoid/jmeter:5.6-plugins-21-jre
```

> Please keep into account that the image should be stored in Docker registry, public or private one. In case of local images, 

Reviewer: ```suggestion
Please remember that the image should be stored in a Docker registry, which can be public or private. In the case of local images, 
```

> you can use [Docker one] (https://www.docker.com/blog/how-to-use-your-own-registry-2/) or something similar.

Reviewer: ```suggestion
you can use this [Docker one] (https://www.docker.com/blog/how-to-use-your-own-registry-2/) or something similar.
```

> At the moment we get the image metadata only from the Docker registry, not container storage or docker-daemon.

Reviewer: ```suggestion
Currently, we receive the image metadata only from the Docker registry, not the container storage or docker-daemon.
```
